src/main/java/ir/sahab/cleanup/Cleanups.java

```diff
@@ -13,9 +12,8 @@
 /**
  * A utility class that helps to free up all desired resources and clean-up operations by a single call.
  *
- * <p>It accepts a list of {@link AutoCloseable}(s) and {@link RunnableWithException} statements.
- * Upon calling {@link #doAll()} all {@link AutoCloseable}(s) will be called and
- * all {@link RunnableWithException} statements will be executed.
+ * <p>It accepts a list of {@link AutoCloseable}. Upon calling {@code #doAll()}
```

Comment:
Replace:
{@link AutoCloseable} > {@link AutoCloseable}s and Lamda callbacks.

Although technically, they are the same, mentioning it helps the reader.